src/app/ui/NewTaskForm.tsx

```diff
@@ -0,0 +1,146 @@
+'use client'
+
+import { PrimaryBtn } from '@/components/buttons/PrimaryBtn'
+import { SecondaryBtn } from '@/components/buttons/SecondaryBtn'
+import { SelectorType } from '@/components/inputs/Selector'
+import Selector from '@/components/inputs/Selector'
+import { StyledTextField } from '@/components/inputs/TextField'
+import { AppMargin, SizeofAppMargin } from '@/hoc/AppMargin'
+import { AttachmentIcon } from '@/icons'
+import { setShowModal } from '@/redux/features/createTaskSlice'
+import store from '@/redux/store'
+import { StatusKey, statusIcons } from '@/utils/iconMatcher'
+import { Close } from '@mui/icons-material'
+import { Avatar, Box, Stack, Typography, styled } from '@mui/material'
+import { ReactNode } from 'react'
+import { status, assignee } from '@/utils/mockData'
+import { IAssignee } from '@/types/interfaces'
+import { useHandleSelectorComponent } from '@/hooks/useHandleSelectorComponent'
+
+export const NewTaskForm = () => {
+  const { renderingItem: statusValue, updateRenderingItem: updateStatusValue } = useHandleSelectorComponent({
+    item: status[0],
+  })
+  const { renderingItem: assigneeValue, updateRenderingItem: updateAssigneeValue } = useHandleSelectorComponent({
+    item: assignee[0],
+  })
+
+  return (
+    <NewTaskContainer>
+      <Stack
+        direction="row"
+        alignItems="center"
+        justifyContent="end"
+        sx={{
+          border: (theme) => `1px solid ${theme.color.borders.borderDisabled}`,
+        }}
+      >
+        <AppMargin size={SizeofAppMargin.MEDIUM} py="12px">
+          <Close
```

Comment:
Please reference this comment https://github.com/copilot-platforms/tasks-app/pull/5#pullrequestreview-2000955150

Reply:
Sounds great!